*** planner-multi.el.original	Thu Aug  4 17:49:07 2005
--- planner-multi.el	Thu Aug  4 17:44:47 2005
***************
*** 262,268 ****
         ;; Set up the new links list for easy testing
         (setq new-pages (mapcar 'emacs-wiki-wiki-base (planner-multi-split page-name)))
          ;; Adjust the future page name if we're coming from a task with a date
!        (unless (member date new-pages)
  	 (setq page-name 
  	       (concat date planner-multi-separator page-name))
  	 (setq new-pages (cons date new-pages)))
--- 262,269 ----
         ;; Set up the new links list for easy testing
         (setq new-pages (mapcar 'emacs-wiki-wiki-base (planner-multi-split page-name)))
          ;; Adjust the future page name if we're coming from a task with a date
!        (unless (or (null date)
! 		   (member date new-pages))
  	 (setq page-name 
  	       (concat date planner-multi-separator page-name))
  	 (setq new-pages (cons date new-pages)))
