diff --new-file --recursive --context orig/LICENSE new/LICENSE
*** orig/LICENSE	Wed Dec 31 19:00:00 1969
--- new/LICENSE	Thu Mar 19 15:13:36 2009
***************
*** 0 ****
--- 1,47 ----
+ Availability of SM
+ 
+ System Requirements:
+ 
+ SM runs on Unix and Unix-variant systems (yes, including Linux) and
+ VMS systems as well as on DOS PCs, under Windows (using Borland C++
+ v3), and under OS/2. It also runs on Mac OSX but not on previous
+ versions of MacOS (A/UX works, however). SM has drivers for Sunview,
+ any form of X11, SGI, various tek401x emulators, VMS UIS, LN03, Imagen
+ impress, QMS quic, HP Laserjet, and Postscript laser printers (also
+ color and encapsulated postscript), raster devices, HPGL, REGIS
+ graphics, and several odd (and likely obsolete) graphics
+ terminals. The PC implementation uses Borland's BGI drivers, so works
+ for any devices supported by that library. 
+ 
+ Cost:
+ 
+ SM is available for $300 U.S., $450 Canadian to university
+ departments, or small colleges. A university-wide site license is
+ $1500 U.S., $2250 Canadian. Prices for commercial or government
+ agencies are $500 U.S. for a department, $2500 U.S. for an entire
+ site. 
+ 
+ You only buy it once - the license entitles you to unlimited free
+ upgrades. The distribution includes the source, documentation (TeX
+ files), and permission to install the program on all the machines at
+ your site. 
+ 
+ Payment is by check or money order in US dollars made payable to
+ Patricia Monger, or by bank transfer. There is a $35 additional charge
+ to cover bank fees if you pay by bank transfer (unless you are able to
+ arrange through your bank that all fees will be covered by you), or if
+ the check does not have the standard bank MICR encoding on it. 
+ 
+ Restrictions:
+ 
+ SM is not for sale to groups who want to use it for weapons related
+ research projects for the military agencies of any country. 
+ 
+ Distribution:
+ 
+ SM is distributed by ftp. If you absolutely must have a tape or floppy
+ we can probably arrange it. 
+ 
+ Contact:
+ If you have further questions, please send mail to <monger@mcmaster.ca>.
+ 
diff --new-file --recursive --context orig/Makefile.skl new/Makefile.skl
*** orig/Makefile.skl	Wed Nov  1 10:47:51 2006
--- new/Makefile.skl	Thu Mar 19 15:17:33 2009
***************
*** 41,53 ****
  #	DESTMAN: manual page(s)
  #	DESTLISP: emacs lisp files
  #
! DEST = /usr/local
! DESTDIR = $(DEST)/bin
! DESTLIB = $(DEST)/lib
! DESTETC = $(DEST)/lib/sm
! DESTINFO = $(DEST)/info
! DESTMAN = $(DEST)/man/man1
! DESTLISP = $(DEST)/emacs/lisp
  #
  # Suffix for SM libraries; usually "" if REAL is float, _d if double
  #
--- 41,67 ----
  #	DESTMAN: manual page(s)
  #	DESTLISP: emacs lisp files
  #
! MACPORTSPREFIX=/sw
! MACPORTSBIN=bin
! MACPORTSLIB=lib
! MACPORTSETC=lib/sm
! MACPORTSINFO=share/info
! MACPORTSMAN=share/man/man1
! MACPORTSLISP=share/emacs/site-lisp
! CFGPREFIX=/sw
! CFGBIN=$(CFGPREFIX)/bin
! CFGLIB=$(CFGPREFIX)/lib
! CFGETC=$(CFGPREFIX)/lib/sm
! CFGINFO=$(CFGPREFIX)/share/info
! CFGMAN=$(CFGPREFIX)/share/man/man1
! CFGLISP=$(CFGPREFIX)/share/emacs/site-lisp
! DEST = $MACPORTSPREFIX
! DESTDIR = $(MACPORTSPREFIX)/$(MACPORTSBIN)
! DESTLIB = $(MACPORTSPREFIX)/$(MACPORTSLIB)
! DESTETC = $(MACPORTSPREFIX)/$(MACPORTSETC)
! DESTINFO = $(MACPORTSPREFIX)/$(MACPORTSINFO)
! DESTMAN = $(MACPORTSPREFIX)/$(MACPORTSMAN)
! DESTLISP = $(MACPORTSPREFIX)/$(MACPORTSLISP)
  #
  # Suffix for SM libraries; usually "" if REAL is float, _d if double
  #
***************
*** 110,119 ****
  		if test "$$f" = ""; then continue; fi; \
  		fd=`echo $$f | sed -e 's#/[^/]*$$##'`; \
  		if test ! -d $$fd ; then \
! 			echo creating $$fd; mkdir $$fd; \
  		fi ; \
  		if test ! -d $$f ; then \
! 			echo creating $$f; mkdir $$f; \
  		fi; \
  	done
  	cp src/sm rasterise make_tags $(DESTDIR)
--- 124,133 ----
  		if test "$$f" = ""; then continue; fi; \
  		fd=`echo $$f | sed -e 's#/[^/]*$$##'`; \
  		if test ! -d $$fd ; then \
! 			echo creating $$fd; mkdir -p $$fd; \
  		fi ; \
  		if test ! -d $$f ; then \
! 			echo creating $$f; mkdir -p $$f; \
  		fi; \
  	done
  	cp src/sm rasterise make_tags $(DESTDIR)
***************
*** 142,176 ****
  		echo "graphcap.local exists; I am not modifying it"; \
  	else \
  		echo "	Setting up graphcap.local"; \
! 		echo "BIN = $(DESTDIR)" > $(DESTETC)/graphcap.local; \
  	fi
  	@ if test ! -f /etc/termcap; then cp termcap $(DESTETC); fi
  	@- echo "	Copying macro and help directories"
  	@ for f in macro help; do \
  		if test ! -d $(DESTETC)/$$f ; then \
! 			mkdir $(DESTETC)/$$f ; \
  		fi ; \
  		cp `find $$f ! -type d -print` $(DESTETC)/$$f ; \
  	done
  	cp sm.man $(DESTMAN)/sm.1
! 	@ if test ! -d $(DESTETC) ; then mkdir $(DESTETC); fi
  	@- if test -f $(DESTETC)/.sm; then \
  		echo File $(DESTETC)/.sm exists\; I am not modifying it; \
  	   else \
  		( \
! 		echo "fonts		//fonts.bin" ;\
  		echo "history_file	.smhist" ;\
  		echo "history		80" ;\
  		echo "temp_dir	/tmp/" ;\
! 		echo "+graphcap	//graphcap.local" ;\
  		for f in filecap graphcap ; do \
! 			echo "$$f		//$$f" ;\
  		done;\
  		if test ! -f /etc/termcap; then \
! 			echo "termcap		//termcap" ;\
  		fi; \
  		for f in help macro ; do \
! 			echo "$$f		//$$f/" ;\
  		done;\
  		) > $(DESTETC)/.sm; \
  	   fi
--- 156,190 ----
  		echo "graphcap.local exists; I am not modifying it"; \
  	else \
  		echo "	Setting up graphcap.local"; \
! 		echo "BIN = $(CFGBIN)" > $(DESTETC)/graphcap.local; \
  	fi
  	@ if test ! -f /etc/termcap; then cp termcap $(DESTETC); fi
  	@- echo "	Copying macro and help directories"
  	@ for f in macro help; do \
  		if test ! -d $(DESTETC)/$$f ; then \
! 			mkdir -p $(DESTETC)/$$f ; \
  		fi ; \
  		cp `find $$f ! -type d -print` $(DESTETC)/$$f ; \
  	done
  	cp sm.man $(DESTMAN)/sm.1
! 	@ if test ! -d $(DESTETC) ; then mkdir -p $(DESTETC); fi
  	@- if test -f $(DESTETC)/.sm; then \
  		echo File $(DESTETC)/.sm exists\; I am not modifying it; \
  	   else \
  		( \
! 		echo "fonts		$(CFGETC)/fonts.bin" ;\
  		echo "history_file	.smhist" ;\
  		echo "history		80" ;\
  		echo "temp_dir	/tmp/" ;\
! 		echo "+graphcap	$(CFGETC)/graphcap.local" ;\
  		for f in filecap graphcap ; do \
! 			echo "$$f		$(CFGETC)/$$f" ;\
  		done;\
  		if test ! -f /etc/termcap; then \
! 			echo "termcap		$(CFGETC)/termcap" ;\
  		fi; \
  		for f in help macro ; do \
! 			echo "$$f		$(CFGETC)/$$f/" ;\
  		done;\
  		) > $(DESTETC)/.sm; \
  	   fi
diff --new-file --recursive --context orig/edit_options_h new/edit_options_h
*** orig/edit_options_h	Wed Dec 31 19:00:00 1969
--- new/edit_options_h	Thu Mar 19 15:13:36 2009
***************
*** 0 ****
--- 1,22 ----
+ #!/usr/bin/perl -i
+ $fname = "options_h_message.txt";
+ $output = 0;
+ open MESSAGE, "> $fname" || die "Cannot open file: $!\n";
+ 
+ while (<>) {
+   if (/\/\*/) {
+     $output = 1;
+   }
+   if ($output == 0) {
+     print MESSAGE;
+   }
+   if ($output == 1) {
+     print;
+   }
+ }
+ 
+ close MESSAGE;
+ 
+ #           open(PRINTER, "| lpr -Plp1")    || die "cannot fork: $!";
+ #           print PRINTER "stuff\n";
+ #           close(PRINTER)                  || die "can't close lpr: $!";
diff --new-file --recursive --context orig/get_response new/get_response
*** orig/get_response	Wed Dec 31 19:00:00 1969
--- new/get_response	Thu Mar 19 15:13:36 2009
***************
*** 0 ****
--- 1,23 ----
+ #!/usr/bin/perl
+ $fname = "options_h_message.txt";
+ 
+ open MESSAGE, "$fname";
+ print "**********************************\n";
+ while (<MESSAGE>) {
+   print;
+ }
+ close MESSAGE;
+ print "**********************************\n";
+ print "A MESSAGE FROM THE SM PACKAGE MAINTAINER"
+ print "Please see the above message from the SM authors.\n";
+ print "In order to make the macports install seamless, I have modified\n";
+ print "the options.h. file, but in order to preserve their rights\n";
+ print "please type \"yes\" and press enter to legally indicate that\n";
+ print "you have done what they've asked.\n";
+ print "? ";
+ $response = <>;
+ $response =~ s/\W//g;
+ $response =~ tr/A-Z/a-z/;
+ if ($response ne "yes") {
+     exit 1;
+ }
diff --new-file --recursive --context orig/set_opts new/set_opts
*** orig/set_opts	Wed Nov  1 09:34:37 2006
--- new/set_opts	Thu Mar 19 15:13:36 2009
***************
*** 785,795 ****
  		dest=$reply
  	fi
  fi
  destbin=$dest/bin
  destlib=$dest/lib
  destetc=$destlib/sm
! destinfo=$dest/info
! destman=$dest/man/man1
  #
  while [ "$destflag" != "1" ]; do
  	echo "The commands make install (and make installinfo)"\
--- 785,802 ----
  		dest=$reply
  	fi
  fi
+ macportsprefix=$dest
+ macportsbin=bin
+ macportslib=lib
+ macportsetc=$destlibmacports/sm
+ macportsinfo=share/info
+ macportsman=share/man/man1
+ macportslisp=share/emacs/site-lisp
  destbin=$dest/bin
  destlib=$dest/lib
  destetc=$destlib/sm
! destinfo=$dest/share/info
! destman=$dest/share/man/man1
  #
  while [ "$destflag" != "1" ]; do
  	echo "The commands make install (and make installinfo)"\
***************
*** 833,842 ****
  	destlisp=""
  	;;
    *)
! 	destlisp="$dest/emacs/site-lisp"
! 	echo $n "In which directory? [$destlisp] "
! 	read reply
! 	if [ "$reply" != "" ]; then destlisp=$reply; fi
  	;;
  esac
  #
--- 840,849 ----
  	destlisp=""
  	;;
    *)
! 	destlisp="$dest/share/emacs/site-lisp"
! #	echo $n "In which directory? [$destlisp] "
! #	read reply
! #	if [ "$reply" != "" ]; then destlisp=$reply; fi
  	;;
  esac
  #
***************
*** 1021,1036 ****
  #
  # Start with Makefile
  #
  echo "Creating Makefile..."
  sed	-e "s#^CC = .*#CC = $cc#" \
  	-e "s|^CFLAGS = .*|CFLAGS = $cflags -I$incfiles|" \
! 	-e "s|^DEST = .*|DEST = $dest|" \
! 	-e "s|^DESTDIR =.*|DESTDIR = $destbin|" \
! 	-e "s|^DESTLIB =.*|DESTLIB = $destlib|" \
! 	-e "s|^DESTETC =.*|DESTETC = $destetc|" \
! 	-e "s|^DESTINFO =.*|DESTINFO = $destinfo|" \
! 	-e "s|^DESTLISP =.*|DESTLISP = $destlisp|" \
! 	-e "s|^DESTMAN =.*|DESTMAN = $destman|" \
  	-e "s|^LIBSUFFIX =.*|LIBSUFFIX = $libsuffix|" \
  	-e "$choose_parser" \
  	-e "$sed_build_swig" \
--- 1028,1051 ----
  #
  # Start with Makefile
  #
+ # cut lines from below
+ #	-e "s|^DEST = .*|DEST = $dest|" \
+ #	-e "s|^DESTDIR =.*|DESTDIR = $destbin|" \
+ #	-e "s|^DESTLIB =.*|DESTLIB = $destlib|" \
+ #	-e "s|^DESTETC =.*|DESTETC = $destetc|" \
+ #	-e "s|^DESTINFO =.*|DESTINFO = $destinfo|" \
+ #	-e "s|^DESTMAN =.*|DESTMAN = $destman|" \
+ #	-e "s|^DESTLISP =.*|DESTLISP = $destlisp|" \
  echo "Creating Makefile..."
  sed	-e "s#^CC = .*#CC = $cc#" \
  	-e "s|^CFLAGS = .*|CFLAGS = $cflags -I$incfiles|" \
! 	-e "s|^MACPORTSPREFIX = .*|MACPORTSPREFIX = $macportsprefix|" \
! 	-e "s|^MACPORTSBIN = .*|MACPORTSBIN = $macportsbin|" \
! 	-e "s|^MACPORTSLIB = .*|MACPORTSLIB = $macportslib|" \
! 	-e "s|^MACPORTSETC = .*|MACPORTSETC = $macportsetc|" \
! 	-e "s|^MACPORTSINFO = .*|MACPORTSINFO = $macportsinfo|" \
! 	-e "s|^MACPORTSMAN = .*|MACPORTSMAN = $macportsman|" \
! 	-e "s|^MACPORTSLISP = .*|MACPORTSLISP = $macportslisp|" \
  	-e "s|^LIBSUFFIX =.*|LIBSUFFIX = $libsuffix|" \
  	-e "$choose_parser" \
  	-e "$sed_build_swig" \
