diff --recursive --context pyIDL-0.5.orig/__init__.py pyIDL-0.5/__init__.py
*** pyIDL-0.5.orig/__init__.py	Wed Apr 18 11:07:22 2007
--- pyIDL-0.5/__init__.py	Tue May  8 12:17:34 2007
***************
*** 2,13 ****
--- 2,16 ----
  # 
  # Michael McKerns
  # mmckerns@caltech.edu 
+ import os
  from rsiidl import __doc__ as idldoc
  __doc__ = idldoc
  
  def idl(stdout=True):
      """idl([stdout]): get help with 'help(pyIDL)'"""
      from rsiidl import idl as idlFactory
+     os.environ['IDL_DIR']='/Applications/rsi/idl_6.1'
+     os.environ['IDL_DIR']='/Applications/rsi/idl_6.1'
      return idlFactory(stdout)
  
  def copyright():
diff --recursive --context pyIDL-0.5.orig/module/pyIDL.cc pyIDL-0.5/module/pyIDL.cc
*** pyIDL-0.5.orig/module/pyIDL.cc	Wed Apr 18 11:07:37 2007
--- pyIDL-0.5/module/pyIDL.cc	Tue May  8 12:15:37 2007
***************
*** 391,397 ****
--- 391,402 ----
    idl_ArrayObject *arr;
    IDL_VPTR var;
    char *data, *name;
+ #if (PY_VERSION_HEX < 0x02050000)
    int len, type, nd, i;
+ #else
+   int type, nd, i;
+   Py_ssize_t len; 
+ #endif
    IDL_MEMINT *dim;
    if (!PyArg_ParseTuple(args, "sOiO", &name, &buf, &type, &shape)) return NULL;
    if (PyObject_AsWriteBuffer(buf, (void **)(&data), &len) == -1) return NULL;
diff --recursive --context pyIDL-0.5.orig/setup.py pyIDL-0.5/setup.py
*** pyIDL-0.5.orig/setup.py	Wed Apr 18 11:07:22 2007
--- pyIDL-0.5/setup.py	Tue May  8 12:17:34 2007
***************
*** 45,58 ****
          IDLLIBS = ['idl','Xm','Xp','Xpm','Xmu','Xext','Xt','SM','ICE',
                     'Xinerama','X11','dl','termcap','rt','m','pthread','gcc_s'] 
      else: #6.0 to 6.2
!         IDLLIBS = ['idl','Xm','Xp','Xpm','Xext','Xt','SM','ICE','X11',
                     'dl','termcap','rt','m','pthread'] 
  
      if platform[:6] == 'darwin':
          IDLLIBS.remove('rt')
!         IDLLIBS.append('MesaGL6_2')
!         IDLLIBS.append('MesaGLU6_2')
!         IDLLIBS.append('OSMesa6_2')
          IDLLIBS.append('freetype2_1_3')
  
      module1 = Extension('_pyIDLmodule',
--- 45,58 ----
          IDLLIBS = ['idl','Xm','Xp','Xpm','Xmu','Xext','Xt','SM','ICE',
                     'Xinerama','X11','dl','termcap','rt','m','pthread','gcc_s'] 
      else: #6.0 to 6.2
!         IDLLIBS = ['idl','Xm','Xmu','Xp','Xpm','Xext','Xt','SM','ICE','X11',
                     'dl','termcap','rt','m','pthread'] 
  
      if platform[:6] == 'darwin':
          IDLLIBS.remove('rt')
!         IDLLIBS.append('MesaGL4_0')
!         IDLLIBS.append('MesaGLU4_0')
!         IDLLIBS.append('OSMesa4_0')
          IDLLIBS.append('freetype2_1_3')
  
      module1 = Extension('_pyIDLmodule',
