diff --recursive --context PyX-0.10.orig/manual/graph.tex PyX-0.10/manual/graph.tex
*** PyX-0.10.orig/manual/graph.tex	Wed Oct  3 14:53:43 2007
--- PyX-0.10/manual/graph.tex	Wed Apr  2 00:00:49 2008
***************
*** 42,48 ****
  
  \begin{figure}[ht]
  \centerline{\includegraphics{graph}}
! \caption{A minimalistic plot for the data from file \file{graph.dat}.}
  \label{fig:graph}
  \end{figure}
  
--- 42,48 ----
  
  \begin{figure}[ht]
  \centerline{\includegraphics{graph}}
! \caption{A minimalistic plot for the data from file \textit{graph.dat}.}
  \label{fig:graph}
  \end{figure}
  
diff --recursive --context PyX-0.10.orig/pyx/config.py PyX-0.10/pyx/config.py
*** PyX-0.10.orig/pyx/config.py	Wed May 24 00:30:54 2006
--- PyX-0.10/pyx/config.py	Wed Apr  2 00:00:20 2008
***************
*** 23,29 ****
  import ConfigParser, os.path
  import siteconfig
  
! cflist = [os.path.join(siteconfig.pyxrcdir, "pyxrc"),  os.path.expanduser("~/.pyxrc")]
  
  config = ConfigParser.ConfigParser()
  config.read(cflist)
--- 23,29 ----
  import ConfigParser, os.path
  import siteconfig
  
! cflist = [os.path.join(siteconfig.pyxrcdir, "pyxrc-py@PYTHON_FLAVOR@"),  os.path.expanduser("~/.pyxrc")]
  
  config = ConfigParser.ConfigParser()
  config.read(cflist)
diff --recursive --context PyX-0.10.orig/setup.cfg PyX-0.10/setup.cfg
*** PyX-0.10.orig/setup.cfg	Fri Apr 28 03:45:09 2006
--- PyX-0.10/setup.cfg	Tue Apr  1 23:56:29 2008
***************
*** 2,12 ****
  # In this section you can specify which c extension modules should be built
  
  # C extension module for fast t1font decoding and encoding
! build_t1code=0
  
  # Python bindings for the kpathsea library. You need the kpathsea header
  # and library and you may need to specify their location below.
! build_pykpathsea=0
  
  [build_ext]
  # additional include and library directories:
--- 2,12 ----
  # In this section you can specify which c extension modules should be built
  
  # C extension module for fast t1font decoding and encoding
! build_t1code=1
  
  # Python bindings for the kpathsea library. You need the kpathsea header
  # and library and you may need to specify their location below.
! build_pykpathsea=1
  
  [build_ext]
  # additional include and library directories:
***************
*** 14,18 ****
  #   the pykpathsea extention module and libkpathsea is not installed in
  #   the default search paths of your compiler and linker
  
! # include_dirs=
! # library_dirs=
--- 14,18 ----
  #   the pykpathsea extention module and libkpathsea is not installed in
  #   the default search paths of your compiler and linker
  
! include_dirs=@PREFIX@/include
! library_dirs=@PREFIX@/lib
diff --recursive --context PyX-0.10.orig/setup.py PyX-0.10/setup.py
*** PyX-0.10.orig/setup.py	Wed May 24 06:33:56 2006
--- PyX-0.10/setup.py	Tue Apr  1 23:59:16 2008
***************
*** 61,67 ****
      data_files.append((dir, files))
      siteconfignames[files] = siteconfigname
  
! adddatafiles("lfsdir", "share/pyx", ["pyx/lfs/10pt.lfs",
                                       "pyx/lfs/11pt.lfs",
                                       "pyx/lfs/12pt.lfs",
                                       "pyx/lfs/10ptex.lfs",
--- 61,67 ----
      data_files.append((dir, files))
      siteconfignames[files] = siteconfigname
  
! adddatafiles("lfsdir", "share/pyx-py@PYTHON_FLAVOR@", ["pyx/lfs/10pt.lfs",
                                       "pyx/lfs/11pt.lfs",
                                       "pyx/lfs/12pt.lfs",
                                       "pyx/lfs/10ptex.lfs",
***************
*** 71,81 ****
                                       "pyx/lfs/foils20pt.lfs",
                                       "pyx/lfs/foils25pt.lfs",
                                       "pyx/lfs/foils30pt.lfs"])
! adddatafiles("sharedir", "share/pyx", ["contrib/pyx.def"])
  
  # Note that on windows we can't install to absolute paths. Hence
  # we put the global pyxrc into the share directory as well.
! adddatafiles("pyxrcdir", os.name != "nt" and "/etc" or "share/pyx", ["pyxrc"])
  
  ################################################################################
  # extend install commands to overwrite siteconfig.py during build and install
--- 71,81 ----
                                       "pyx/lfs/foils20pt.lfs",
                                       "pyx/lfs/foils25pt.lfs",
                                       "pyx/lfs/foils30pt.lfs"])
! adddatafiles("sharedir", "share/pyx-py@PYTHON_FLAVOR@", ["contrib/pyx.def"])
  
  # Note that on windows we can't install to absolute paths. Hence
  # we put the global pyxrc into the share directory as well.
! adddatafiles("pyxrcdir", os.name != "nt" and "etc" or "share/pyx-py@PYTHON_FLAVOR@", ["pyxrc-py@PYTHON_FLAVOR@"])
  
  ################################################################################
  # extend install commands to overwrite siteconfig.py during build and install
***************
*** 123,129 ****
              # (the following four lines are copied from within the install_data.run loop)
              dir = convert_path(dir)
              if not os.path.isabs(dir):
!                 dir = os.path.join(self.install_dir, dir)
              elif self.root:
                  dir = change_root(self.root, dir)
  
--- 123,129 ----
              # (the following four lines are copied from within the install_data.run loop)
              dir = convert_path(dir)
              if not os.path.isabs(dir):
!                 dir = os.path.join("@PREFIX@", dir)
              elif self.root:
                  dir = change_root(self.root, dir)
  
