diff --context unionfs-fuse-0.18.orig/Makefile unionfs-fuse-0.18/Makefile
*** unionfs-fuse-0.18.orig/Makefile	Thu Aug  9 04:58:38 2007
--- unionfs-fuse-0.18/Makefile	Tue Apr  1 21:00:22 2008
***************
*** 2,8 ****
  CPPFLAGS += -D_FILE_OFFSET_BITS=64 -DFUSE_USE_VERSION=26
  LDFLAGS += 
  
! LIB = -lfuse -lpthread -lm -lrt
  
  HASHTABLE_OBJ = hashtable.o hashtable_itr.o hash.o elfhash.o
  UNIONFS_OBJ = unionfs.o stats.o opts.o debug.o findbranch.o readdir.o general.o unlink.o cow.o cow_utils.o
--- 2,8 ----
  CPPFLAGS += -D_FILE_OFFSET_BITS=64 -DFUSE_USE_VERSION=26
  LDFLAGS += 
  
! LIB = -lfuse -lpthread -lm 
  
  HASHTABLE_OBJ = hashtable.o hashtable_itr.o hash.o elfhash.o
  UNIONFS_OBJ = unionfs.o stats.o opts.o debug.o findbranch.o readdir.o general.o unlink.o cow.o cow_utils.o
diff --context unionfs-fuse-0.18.orig/unionfs.c unionfs-fuse-0.18/unionfs.c
*** unionfs-fuse-0.18.orig/unionfs.c	Thu Aug  9 04:58:38 2007
--- unionfs-fuse-0.18/unionfs.c	Tue Apr  1 21:01:58 2008
***************
*** 157,163 ****
  	int res;
  	if (isdatasync) {
  #ifdef _POSIX_SYNCHRONIZED_IO
! 		res = fdatasync(fi->fh);
  #else
  		res = fsync(fi->fh);
  #endif
--- 157,163 ----
  	int res;
  	if (isdatasync) {
  #ifdef _POSIX_SYNCHRONIZED_IO
! 		res = fsync(fi->fh);
  #else
  		res = fsync(fi->fh);
  #endif
